assets/src/edit-story/components/movable/moveStyle.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+	.default-movable .moveable-control,
+	.default-movable .moveable-line.moveable-rotation-line .moveable-control {
+		background: #1a73e8 !important;
+		border: 2px solid #fff !important;
+		margin-left: -6px;
+		box-shadow: 0 2px 5px rgba(0, 0, 0, 0.15);
+	}
+
+	.default-movable .moveable-direction.moveable-n {
+		display: none !important;
```

Comment:
Why is this?

Reply:
This is coming from the UI mocks, the north anchor is not shown

Reply:
Got it.